components/rhs_header_post/index.ts

```diff
@@ -22,26 +28,29 @@ import {getIsRhsExpanded} from 'selectors/rhs';
 
 import RhsHeaderPost from './rhs_header_post';
 
-function mapStateToProps(state: GlobalState) {
+type OwnProps = Pick<ComponentProps<typeof RhsHeaderPost>, 'rootPostId'>
+
+function mapStateToProps(state: GlobalState, {rootPostId}: OwnProps) {
     return {
         isExpanded: getIsRhsExpanded(state),
         relativeTeamUrl: getCurrentRelativeTeamUrl(state),
+        currentTeamId: getCurrentTeamId(state),
+        currentUserId: getCurrentUserId(state),
+        isCollapsedThreadsEnabled: isCollapsedThreadsEnabled(state),
+        isFollowingThread: getThread(state, rootPostId)?.is_following,
     };
 }
 
-function mapDispatchToProps(dispatch: Dispatch) {
-    return {
-        actions: bindActionCreators({
```

Comment:
Why are we no longer binding the action creators?

Reply:
They're still bound, but with the flattened actions and passing them in, `connect` automatically does the binding.